hack/verify-govet.sh

```diff
@@ -23,8 +27,12 @@ function govet_blacklist_contains() {
 	return 1
 }
 
-for test_dir in $(os::util::list_go_src_dirs); do
-	if ! result="$(go tool vet -shadow=false -printfuncs=Info,Infof,Warning,Warningf "${test_dir}" 2>&1)"; then
+# This is required before we run govet for the results to be correct.
+# See https://github.com/golang/go/issues/16086 for details.
+go install ./cmd/...
+
+for test_dir in $(go list -e ./...); do
```

Comment:
Why is this different from the list func before? Why is it  better?

Reply:
`go list` will give you a list of all dirs where it finds go source files, this includes subdirs as well. Our version of list only lists top-level dirs. I went with this one to ensure all go source files are vetted, additionally there were some weird edge cases when I was passing top-level dirs with `...` at the end.